ci/codespell-ignore-words.txt

```diff
@@ -4,13 +4,18 @@ ba
 cannotation
 coo
 curvelinear
+dedented
+falsy
 flate
+hax
```

Comment:
Use in 

https://github.com/matplotlib/matplotlib/blob/23730b41febeb3cc80bd4733a228f0ed22e1b6d5/examples/lines_bars_and_markers/vline_hline_demo.py#L20

for "horizontal axis" is reasonable, so added. However, if it would be preferable to make the example variables more explicit and verbose I can edit them instead.